src/ssh/HISTORY.md

```diff
@@ -1,5 +1,9 @@
 Release History
 ===============
+1.1.3
+-----
+* Fix bug where extension prints ssh banners after the connection is closed.
```

Comment:
If you want to release a new extension version, please also upgrade the version defined in `setup.py`

Reply:
@zhoxing-ms I was planning on upgrading the version only after all the open PRs for the next release are merged and I have the chance to run some final tests. For now I'll just remove the changes in the HISTORY.md file for this PR.

Reply:
@vthiebaut10 OK. Or I can merge this PR first, and you can add new content to the history notes of subsequent PRs.

Reply:
In the future, if you do not want to release a new version immediately, I suggest you add comments in PR to tell us